src/docs/Welcome.mdx

```diff
@@ -19,64 +19,36 @@ export const versionsHeadersData = [
 export const versionsData = [
   [
     <Text size="body2" whiteSpace="nowrap">
-      Reservoir 3.x
+      Reservoir 4.0.0
     </Text>,
     <Text size="body2" whiteSpace="nowrap">
-      ongoing
+      July 2025
     </Text>,
     <Text size="body2">
-      Reservoir is constantly undergoing improvements with a new version
-      published ever two weeks.
+      Reservoir v4 marked a return to HTML, CSS, and JavaScript fundamentals to
```

Comment:
"a return" somewhat sounds like we left this approach... how about "focuses on..."?

Reply:
Sure! Can update in the mig guide as well.